wpeverest/everest-forms
# Fix - Accepting Numbers Below 0 Despite Minimum Limit of 0.

### All Submissions:


### Changes proposed in this Pull Request:
Previously Accepted Values Below 0 Despite Setting Minimum Value to 0. This PR fixes this issue.

### How to test the changes in this Pull Request:

1. Create a form
2. Sets the minimum value 0.
3. Verify if the value is less than 0.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Accepting Numbers Below 0 Despite Minimum Limit of 0.
